ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -170,7 +199,24 @@ func NewDEXOnboarding(l *load.Load) *DEXOnboarding {
 // the page is displayed.
 // Part of the load.Page interface.
 func (pg *DEXOnboarding) OnNavigatedTo() {
-	pg.showLoader = false
+	ch := pg.dexc.NotificationFeed()
+	// TODO: return note feed?
+	go func() {
+		for {
+			n, ok := <-ch
+			if !ok {
+				return
```

Comment:
This may also be a good place to return the note and set the channel to nil so it is not returned in OnNavigatedFrom().